### PR TITLE
Convert dev scripts to use JSX

### PR DESCRIPTION
**Pull request by whymarrh**
Refs #6291

This PR removes hyperscript from the development scripts.

### CODE REVIEW

**Comment by whymarrh** [comment]
The integration tests will likely fail, I'm using this PR to debug them 🚧

File: development/selector.js [hunk 1]
```diff
@@ -23,20 +15,32 @@ NewComponent.prototype.render = function () {
...
+      onChange={(event) => {
+        const selectedKey = event.target.value
+        const backgroundConnectionModifier = backGroundConnectionModifiers[selectedKey]
+        modifyBackgroundConnection(backgroundConnectionModifier || {})
+        store.dispatch(actions.update(selectedKey))
+        this.setState({ selected: selectedKey })
+      }}
+    >
+      {Object.keys(states).map((stateName, index) => {
+        return <option key={index} value={stateName}>
```
Comment (Gudahtt) [inline development/selector.js]: Should this be wrapped in parentheses?

Comment (whymarrh) [reply development/selector.js]: Yes, thanks, good catch!

**Comment by whymarrh** [comment]
Looking at the bundle locally, it all seems to work. I don't need to touch the existing `test:flat:build:ui` script. 🙏🏽

**Review by Gudahtt — DISMISSED** [review]
LGTM!

**Comment by whymarrh** [comment]
I rebased this and converted `Selector` back to a class component as it uses state (I missed that)

**Review by Gudahtt — APPROVED** [review]
Whoops, I missed that too. Looks good now though.

**Pull request merged**